magic-modules/make-comment.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+set -x
+
+shopt -s dotglob
+cd mm-initial-pr
+ID=$(git config --get pullrequest.id)
+BRANCH=$(git config --get pullrequest.branch)
+echo $BRANCH
+echo $ID
+cat .git/branch
+cd ..
+cp -r magic-modules-out/* magic-modules-comment
```

Comment:
Why do you need another tree? Do you run this on every update from MM PR branch?

Reply:
Don't understand the question.  The `input` and `output` sections of https://concourse.ci/running-tasks.html, plus the non-obvious fact that there's no such thing as an input/output directory means that I need a new directory every time I do anything.

Reply:
Ah, I get it. I guess my initial confusion was with the folder names. The output folder names do not seem to match their behavior. For example the create-pr has input "magic-modules" and output "magic-modules-out".

Maybe if we break the pipeline in more granular steps it would be easier to see. For example:

- sync: _ => magic-modules
- compile: magic-modules => magic-modules-compiled
- create-pr(terraform): magic-modules-compiled => terraform-pr-out
... and so on.

Especially when we have other products in the pipeline it seems we'll have many different steps. A diagram of the folder dependencies (someone said dot file? :)) would help visualize this. Maybe Concourse can generate that? If not a small .dot file?

Reply:
Yeah, you're right, I'm not very neat about the folder names.  That's a small change that I would like to make after this, if that's okay with you.